etc/kayobe/ansible/scripts/nvmemon.sh

```diff
@@ -21,6 +21,43 @@ if ! command -v nvme >/dev/null 2>&1; then
   exit 1
 fi
 
+if ! command -v jq >/dev/null 2>&1; then
```

Comment:
Please can sync in the upstream change here to get rid of the shell script?

https://github.com/prometheus-community/node-exporter-textfile-collector-scripts/commit/a2b43e19be1e64c31b626ca827506977cac93488